core-it-suite/pom.xml

```diff
@@ -159,6 +161,42 @@ under the License.
       </testResource>
     </testResources>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.its</groupId>
+        <artifactId>core-it-bootstrap</artifactId>
```

Comment:
Stupid question: Are other plugins named like this, i.e., not `maven-XXX-plugin`?

Reply:
Renamed